pontoon/base/views.py

```diff
@@ -104,6 +105,17 @@ def locale(request, locale, template='locale.html'):
     return render(request, template, data)
 
 
+def locales(request, template='locales.html'):
```

Comment:
Should probably be consistent with similar view names and call this the singular "locale" instead.